uiowa/uiowa
# Move uiowa_auth out again

Resolves https://github.com/uiowa/uiowa/issues/5300

Tests are running on https://github.com/uiowa/uiowa_auth (but failing)

This branch `undo_uiowa_auth_move` build is passing.

There are DEV deploy errors but they are unrelated to this work

## To Test

On https://default.dev.drupal.uiowa.edu/ ...
- You don't see the core drupal login at /user and instead get sent to the SAML auth
- You can not see things like password change on a user edit form
- Other fields are disabled for your protection
- https://default.dev.drupal.uiowa.edu/admin/config/people/hawkid exists and is populated with our admin mappings
- As itsadmin, you can remove the admin role from your account, logout and then log back in as your hawkid and have your admin role assigned.

Locally run the uiowa_auth tests:
- `ddev ssh`
- `phpunit -c docroot/core/ docroot/modules/uiowa/uiowa_auth/tests`
